publish.sh

```diff
@@ -0,0 +1,18 @@
+#/bin/sh
+
+# User info
+git config --global user.name "Admin"
+git config --global user.email "patternfly@redhat.com"
+
+# Add upstream authentication token
+git remote add upstream https://$AUTH_TOKEN@github.com/dlabrecq/patternfly-org.git
```

Comment:
Will this change to @github.com/patternfly/patternfly-org.git or do we need a bit more testing?